server/gfmd/db_none.c

```diff
@@ -574,6 +574,38 @@ gfarm_none_mdhost_load(void *closure,
 
 /**********************************************************************/
 
+static gfarm_error_t
```

Comment:
move the user_info related functions to just after the gfarm_none_user_load() function,
to match the modified order of struct db_ops.

Reply:
commit [429eb3d](https://github.com/oss-tsukuba/gfarm/pull/1141/commits/429eb3daa151165a09117234d565149a2a1ac4e6) has not applied above change. Could you change it?